src/libgit2/merge.c

```diff
@@ -1220,11 +1220,18 @@ static int merge_diff_mark_similarity_exact(
 
 	/* Build a map of object ids to conflicts */
 	git_vector_foreach(&diff_list->conflicts, i, conflict_src) {
-		/* Items can be the source of a rename iff they have an item in the
+		/* Items can be the source of a rename if they have an item in the
```

Comment:
This is not a typo. Are we changing the logic?

Reply:
No I didn't change the logic, I really thought it was a typo and wanted to fix it while at here.
Should I remove the hunk?

Reply:
Yes please. It's an abbreviation for "if and only if".

Reply:
Ah I didn't know, that's fixed. Thanks.